onmt/models/model_saver.py

```diff
@@ -72,11 +82,12 @@ def save(self, step, moving_average=None):
                 self._rm_checkpoint(todel)
             self.checkpoint_queue.append(chkpt_name)
 
-    def _save(self, step):
+    def _save(self, step, model):
```

Comment:
Why this change here?

Reply:
It's not defined properly in the base class. We call `self._save(step, model)` in `self.save` method.